### ISSUE TITLE
@ApiParam values are not inhereted

### ISSUE THREAD

**Issue by mrj365** [swagger-api/swagger-core#1675]
When I annotate the parameters with @ApiParam in the super class, I do not see its values in the UI. If I annotate the parameters on the implementation with @ApiParam I do see the values.

swagger-jersey2-jaxrs
1.5.7
```
Interface:

public interface ICategoryController {

	@ApiOperation(value = "Get all categories", notes = "Returns a list of categories", response = CategoryResponse.class, responseContainer="List")
	public Response getAllCategories(
				@ApiParam(value = "Start index of category id", allowableValues = "range[1,1000]", required = true) 
				@QueryParam(value="startIndex") int startIndex, 
				@ApiParam(value = "Limit the number of categories returned", allowableValues = "range[1,1000]", required = true)
				@QueryParam(value="size") int size);

}
```
Impl:

```
	@GET
	@Produces("application/json")
	public Response getAllCategories(@QueryParam(value="startIndex") int startIndex, @QueryParam(value="size") int size)
	{	
		List<Category> categoryList = sampleCategoryService.getAllCategories(startIndex, size);		
		return Response.ok(categoryList).build();		
	}
```

![image](https://cloud.githubusercontent.com/assets/5374278/13228376/abfac6fe-d969-11e5-817a-1be43fc1fd66.png)

**Comment by webron** [comment]
Any reason you have the JAX-RS annotations on both the interface and the implementation?

**Comment by mrj365** [comment]
I thought it would help make things more clear. I just tested witout the JAX-RS annotations in the interface and got the same result.

**Comment by webron** [comment]
How about when you put them both in the same place (only)?

**Comment by mrj365** [comment]
If I put @ApiParam and @QueryParam only on the implementation, it works, but I am trying to avoid putting the swagger documentation on the implementation. If I put them both only on the interface I get a wierd result:

Both on interface:

![image](https://cloud.githubusercontent.com/assets/5374278/13228808/bb04488a-d96b-11e5-8533-f289ef232216.png)

**Comment by fehguy** [comment]
I don't think they inherit from the interface to the implementation

**Comment by mrj365** [comment]
Yes, that is why I opened they ticket. I was under the impression that they would. It seems like a bug.

**Comment by mrj365** [comment]
Just to add a little more info, a similar issue was filed with springfox: https://github.com/springfox/springfox/issues/388

**Comment by fehguy** [comment]
got it.  If someone can point us in the direction of inheriting annotations in implementations, it should be easy to fix in the next release.

**Comment by frantuma** [comment]
similar to #1506, fixed in #1747
